packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.stories.ts

```diff
@@ -219,13 +219,86 @@ export const Empty = {
 
 /**
  * This example shows a select with search functionality.
+ * The filtering of the options will be handled automatically by onyx.
+ * _Info: the property `searchTerm` is disabled in this example, as it should stay undefined for the example to work._
  */
-export const WithSearch = {
+export const WithSearch: Story = {
   args: {
     ...Default.args,
     withSearch: true,
   },
-} satisfies Story;
+  decorators: [
+    /**
+     * Decorator to prevent Storybook from setting the searchTerm
+     * which would disable the included filtering by managed search state
+     */
+    (story, ctx) => ({
+      components: { story },
+      setup: () => {
+        watchEffect(() => {
+          ctx.args.searchTerm = undefined;
+          ctx.args.searchTerm;
```

Comment:
Can you add the reactivity comment from below here, too?

Reply:
done :)